### PR TITLE
Patient Details Fixes

### PR DESCRIPTION
**Pull request by ayush-jtg**
tasks completed - 
1. update Medication Details fix
2. Gender Validation Personal Detail
3.added themes for Covid Status and Clinical status
4.Cluster Group appearing as number fix
5. District appearing as number fix

### CODE REVIEW

File: src/app/store.js [hunk 1]
```diff
@@ -9,7 +9,10 @@ const middleware = [thunk];
 const store = createStore(
     rootReducer,
     initialState,
-    compose(applyMiddleware(...middleware))
+    compose(
+    applyMiddleware(...middleware),
+    window.__REDUX_DEVTOOLS_EXTENSION__ && window.__REDUX_DEVTOOLS_EXTENSION__()
```
Comment (gauravksingh) [inline src/app/store.js:14]: If you are adding this, it has to be only in dev environment

File: src/containers/InventoryForm/index.js [hunk 2]
```diff
@@ -59,7 +59,7 @@ export const InventoryForm = (props) => {
             }
             delete initial.name;
             delete initial.type;
-            setInventoryData({ inventoryData: initial });
+            // setInventoryData({ inventoryData: initial });
```
Comment (gauravksingh) [inline src/containers/InventoryForm/index.js:62]: why remove this??

File: src/containers/Patient/FacilityDetails/createUpdateForm/form.js [hunk 3]
```diff
@@ -49,12 +49,15 @@ export default function Form(props) {
 
   const onSelectFacility = (event, value) => {
     const name = "facility"
-    setFieldTouched(name);
+    setFieldTouched(name, true, false);
+    if(value) {
+    setFieldTouched(name, false, true);
```
Comment (gauravksingh) [inline src/containers/Patient/FacilityDetails/createUpdateForm/form.js:54]: indentation??

File: src/screens/PatientDetail.js [hunk 4]
```diff
@@ -101,19 +101,26 @@ class PatientDetail extends Component {
...
-    }
-    if (response.status === true) {
-      this.props.createToastNotification(
-        ToastUtils.toastDict((new Date()).getTime(), "updated", "Successfully updated ", SUCCESS)
-      )
-    } else {
-      this.props.createToastNotification(
-        ToastUtils.toastDict((new Date()).getTime(), "Added", "Some Errors occurs", DANGER)
-      )
+      if (response.status === true) {
```
Comment (gauravksingh) [inline src/screens/PatientDetail.js:115]: No point in checking === true(I do not want to repeat this comment).

Comment (ayush-jtg) [reply src/screens/PatientDetail.js:115]: sure

File: src/constants/app.const.js [hunk 5]
```diff
@@ -537,4 +544,7 @@ export const patient_facility_status_choices = [
       "name": "Facility exists",
       "theme": "success"
   }
-]
\ No newline at end of file
+]
+
+export const FACILITY_EXISTS = 4;
+
```
Comment (gauravksingh) [inline src/constants/app.const.js:550]: extra line

**Pull request merged**